pyproject.toml

```diff
@@ -85,8 +85,10 @@ streaming_extractors = [
     "s3fs"
 ]
 
+
 preprocessing = [
     "scipy",
+    "ibllib>=2.36.0", # for IBL
```

Comment:
@h-mayorquin 

After discussing with Sam, we think that this is not a `preprocessing` dependency per se, since it's only used for testing.

Maybe this extra should be `test_preprocessing` instead? (as we have for `test_extractors`)

Reply:
All right.